### ISSUE TITLE
CMP_SSEVec4f unused class causing issues

### ISSUE THREAD

**Issue by baldurk** [GPUOpen-Tools/compressonator#147]
Hi there, `CMP_SSEVec4f` in cmp_math_vec4.h and the inclusion of `xmmintrin.h` causes issues compiling compressonator on non-x86 (32 and 64-bit) platforms. Since this class seems to be unused in any of the code as far as I can tell, could it be removed?

**Comment by NPCompress** [comment]
@baldurk  Yes it can be removed. All assembly instructions in cmp_core sources will be removed.
